apps/publish/content/common.py

```diff
@@ -719,7 +719,6 @@ def _publish_associated_items(self, original, updates=None):
 
                 if associated_item.get('state') not in PUBLISH_STATES:
                     # This associated item has not been published before
-                    associated_item.get('task', {}).pop('stage', None)
```

Comment:
I wonder if we can break some logic by removing this line.

Reply:
I wonder if the logic from this PR does not conflict with current changes.

Reply:
> I wonder if the logic from this PR does not conflict with current changes.

I have tested [this](https://github.com/superdesk/superdesk-core/commit/493d549b5647359a221863d2b4fbfb99ccf41c9f) PR, and it's not in conflict with the current changes.